tests/ckeditor/basepath.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals beforeEach, describe, it, expect, CKEDITOR, window, document */
+
+'use strict';
+
+beforeEach( function() {
+	// Ensure that no CKEDITOR_BASEPATH global is available.
+	delete window.CKEDITOR_BASEPATH;
+
+	// Remove all script elements from the document.
+	removeScripts();
+} );
+
+describe( 'ckeditor.basePath', function() {
+	it( 'Full URL', function( done ) {
+		CKEDITOR.require( [ 'ckeditor' ], function( CKEDITOR ) {
+			addScript( 'http://bar.com/ckeditor/ckeditor.js' );
+			expect( CKEDITOR._getBasePath() ).equals( 'http://bar.com/ckeditor/' );
```

Comment:
Hum, in fact there are many possible variations: `to.equal()`, `equal()`, `equals()`.

We may agree on best practices, but in any case I think we should not enforce this because it'll be a pain in the ass for contributors to have their stuff rejected because of such things, considering that chai gives all these possibilities and they're all right.

I'll change what we have not to `to.equal()`.